docs/kh1/worlds.md

```
@@ -5,13 +5,13 @@
 | AL | Agrabah
 | AW | Wonderland
 | DC | Disney Castle
-| DH | Dive to Heart
-| DI | Destiny Island
+| DH | Dive to the Heart
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)